src/main/java/com/redhat/devtools/lsp4ij/client/features/LSPCompletionFeature.java

```diff
@@ -165,7 +165,8 @@ public String getItemText(@NotNull CompletionItem item) {
      */
     @Nullable
     public String getTypeText(CompletionItem item) {
-        return item.getDetail();
+        var labelDetails = item.getLabelDetails();
+        return labelDetails != null ? labelDetails.getDescription() : null;

```

Comment:
@Enaium I have tested your PR with other language server like TypeScript, Go etc which doesn't support label detail and the renderer is bad:

![image](https://github.com/user-attachments/assets/d280dce3-4870-4549-a87b-0403bdf299a4)

The `detail` in this case must be displayed on the right (like vscode does). Please change with 

`return labelDetails != null ? labelDetails.getDescription() : item.getDetail();`